src/applications/combined-debt-portal/debt-letters/components/DebtLettersTable.jsx

```diff
@@ -87,7 +87,6 @@ const DebtLettersTable = ({ debtLinks, hasDependentDebts, isError }) => {
           <button
             type="button"
             className="debt-older-letters usa-button-secondary"
-            aria-controls="older-letters-button"
```

Comment:
Should this have been removed?

Reply:
This was removed because the e2e axe test was failing because there is not an element with an id of older-letters-button.